.github/workflows/cd-pypi.yml

```diff
@@ -0,0 +1,11 @@
+name: cd
+
+on:
+  push:
+    tags:
+    - '**'
+
+jobs:
+  deploy:
+    uses: ecmwf-actions/reusable-workflows/.github/workflows/cd-pypi.yml@v2
```

Comment:
We need to use ecmwf org for the reusable-workflows, not ecmwf-actions. It's being ported so let's do it already correctly

Reply:
good catch! Copypasted from old checkout of some repo :D